### DISCUSSION TITLE
Open new window in current directory

### DISCUSSION THREAD

**Discussion by 3N4N** [microsoft/terminal#15511] [Q&A]
For tabs and splits, we have `"command": "duplicateTab"` and `"splitMode": "duplicate"`, which lets us open new tabs and splits in the current directory with the current profile.  There is no such option for new windows.  I understand this may not be as useful, since the existence of tabs reduces if not removes the need for multiple windows, but I assume it's not a huge modification, implementation wise, so worth it.

**Comment by TiLopes** [comment]
@3N4N how do you split and stay in the same directory? Does it work on WSL?
This my split keybind:
```json
{
  "command": {
     "action": "splitPane",
     "split": "right",
     "splitMode": "duplicate"
   },
   "keys": "alt+shift+minus"
}
```

Forget it I found it. For anyone who comes across this issue go to this link and follow the steps: https://learn.microsoft.com/en-us/windows/terminal/tutorials/new-tab-same-directory

**Reply by tianon** [reply]
I don't know why this is marked as the answer -- it's definitely _not_ the answer to the OP's feature request (which as far as I can tell is still not implemented today).

**Reply by tianon** [reply]
(and to be clear, I'm here specifically because I'm that user who usually wants a new window, not a new tab, but I want it in the same directory/profile as my current window)

**Reply by 3N4N** [reply]
Yeah I dunno why I marked it as an answer either.  Nor have I found any solution to this yet.

**Reply by AdrianoCahete** [reply]
Even the `"splitMode": "duplicate"` isn't work anymore (at least on Terminal preview 1.22.3232.0). 😢
